File: src/main/java/net/lomeli/lomlib/client/models/IMeshVariant.java
package net.lomeli.lomlib.client.models;

import net.minecraft.client.renderer.ItemMeshDefinition;

import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public interface IMeshVariant extends IModelHolder {
    @SideOnly(Side.CLIENT)
    ItemMeshDefinition getCustomMesh();
}

File: src/main/kotlin/net/lomeli/lomlib/client/models/IMeshVariant.kt
package net.lomeli.lomlib.client.models

import net.minecraft.client.renderer.ItemMeshDefinition
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

interface IMeshVariant : IModelHolder{
    @SideOnly(Side.CLIENT) fun getCustomMesh(): ItemMeshDefinition
}